package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wet-boew",
-  "version": "4.0.27-development",
+  "version": "4.0.28-development",
```

Comment:
Aside: This probably should get updated next time as part of the release process